lib/fp.js

```diff
@@ -37,23 +34,21 @@ const compose = (
   return res;
 };
 
-const maybe = (
-  // Apply given function to value or default value
-  fn, // function
-  defVal, // default value
-  value // value (optional)
-  // Returns: result of `fn` or `defVal`
-) => (
+// Apply given function to value or default value
+//   fn - <function>
+//   defVal - default value
+//   value - value (optional)
```

Comment:
I propose to use square brackets around the argument name to convey 'optionality' of the argument. i.e. `[value] - <any>, value`?

Also this doesn't have types for values.